x-pack/legacy/plugins/apm/common/service_map_constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const SERVICE_MAP_TASK_TYPE = 'apmServiceMap';
+export const SERVICE_MAP_TASK_ID = 'apm-servicemap-processor';
```

Comment:
Should we have a convention around camelCase vs kebab-case for constants?

Reply:
We already use snake case for constants (eg `SERVICE_MAP_TASK_ID`) as for the value, it's  depends on the context. There's no existing convention for task types and task ids. Some plugins mix snake and kebab case for id: `plugin_task-id`. from what i see, it looks like other plugins tend to use snake case for the task type. We could change the type to `apm_service_map` to conform if we want to. The task type is represented in the API by an object key in the task definition, so i don't recommend kebab case.